doc/api/configuration.md

```diff
@@ -69,20 +69,18 @@ If you have autopandas set to true, the displaylimit option will not apply. You
 %config SqlMagic.feedback = 0
 ```
 
-## `displaycon`
+## `autocommit`
```

Comment:
looks like you ordered them alphabetically (?)

Reply:
Yes

Reply:
should the ordering be handled in another PR? It's a bit difficult to read the new ones added @edublancas

Reply:
I'll do that